lib/core/dispatchRequest.js

```diff
@@ -22,7 +22,7 @@ function throwIfCancellationRequested(config) {
  * @param {object} config The config that is to be used for the request
  * @returns {Promise} The Promise to be fulfilled
  */
-module.exports = function dispatchRequest(config) {
+module.exports=function dispatchRequest(config) {
```

Comment:
format: style mismatch:
```suggestion
module.exports = function dispatchRequest(config) {
```

`2f4f64f1` ⏎ at column 4 should be removed.
`4e4e06c5` ␣ should be inserted at column 15.
`2f9d4b4e` ␣ should be inserted at column 16.